apps/web/scripts/kernel/handlers/solanaHandler.ts

```diff
@@ -95,15 +95,18 @@ export const signTransactionAbstractionFee: HandleMethod = async ({
 	const txStr = encode(transaction.serialize());
 
 	try {
-		const res = await fetch(`${GASILON_ENDPOINT}/solana/transfer`, {
-			method: 'POST',
-			headers: {
-				'Content-Type': 'application/json',
+		const res = await fetch(
+			`${modules.config.GASILON_ENDPOINT}/solana/transfer`,
+			{
+				method: 'POST',
+				headers: {
+					'Content-Type': 'application/json',
+				},
+				body: JSON.stringify({
+					transaction: txStr,
+				}),
 			},
-			body: JSON.stringify({
-				transaction: txStr,
-			}),
-		});
+		);
```

Comment:
is it reusable for both web and mobile?

Reply:
if we have this function in package app, prefer removing it

Reply:
> if we have this function in package app, prefer removing it

you said that you need to refactor request message on web platform so that i cannot remove this file immediately